redeem/gcodes/M2x.py

```diff
@@ -367,15 +385,17 @@ class M27(M2X):
     # FIXME : adjust current_line_count by the number of commands in the path planner buffer
 
     def execute(self, g):
-        message = " file '{}' printing: {} of {} lines"
 
         self.printer.sd_card_manager.current_lock.acquire()
-        message = message.format(self.printer.sd_card_manager.current_file,
-                                 self.printer.sd_card_manager.current_line_count,
-                                 self.printer.sd_card_manager.current_file_count)
+	current_line_count = self.printer.sd_card_manager.current_line_count
+	current_file_count = self.printer.sd_card_manager.current_file_count
         self.printer.sd_card_manager.current_lock.release()
 
-        self.printer.send_message(g.prot, message)
+	if current_line_count is None or current_file_count is None:
+		return
+
+        message = "SD printing byte {}/{}".format(current_line_count, current_file_count)
```

Comment:
You're counting lines but calling them bytes?

Reply:
The expected format for octoprint is bytes, but we don't know how many bytes each line is. This allows us to still get a progress report on the print status. It's not ideal and if you have a way of getting bytes from a given line without re-parsing each line in M27 I'd love to hear it.